### PR TITLE
[cdc] Time related computed functions support to handle epoch time

### PR DESCRIPTION
**Pull request by yuzelin**
### Purpose

Current time expressions can only handle DATE and TIMESTAMP.
This PR aims to support handle numeric value as epoch time.

### API and Format
For example:
1. year: year(epoch_col)   year(epoch_col, time-unit)
2. date_format: date_format(epoch_col, yyyy-MM-dd)   date_format(epoch_col, yyyy-MM-dd, time-unit)
time-unit: second (default), millis, micros, nanos

### Documentation

CDC Ingestion -> Computed Functions

### CODE REVIEW

**Review by zhuangchong — APPROVED** [review]
+1

File: docs/layouts/shortcodes/generated/time_functions.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+<table class="configuration table table-bordered">
+    <thead>
+    <tr>
+        <th class="text-left" style="width: 30%">Function</th>
+        <th class="text-left" style="width: 70%">Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><h5>year(time-column [, time-unit])</h5></td>
```
Comment (JingsongLi) [inline docs/layouts/shortcodes/generated/time_functions.html:29]: Can you find some other systems to refer to?

Comment (yuzelin) [reply docs/layouts/shortcodes/generated/time_functions.html]: Refer to Flink `TO_TIMESTAMP_LTZ(numeric, precision)`, using precision as argument.

[Temporal Functions](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/functions/systemfunctions/#temporal-functions)

**Comment by JingsongLi** [comment]
+1

**Pull request merged**